src/server/transaction/server/testing/server_test.go

```diff
@@ -119,25 +120,29 @@ func TestTransactions(suite *testing.T) {
 		t.Parallel()
 		ctx := pctx.TestContext(t)
 		env := realenv.NewRealEnv(ctx, t, dockertestenv.NewTestDBConfig(t))
-
+		project := testutil.UniqueString("p-")
 		txn, err := env.PachClient.StartTransaction()
 		require.NoError(t, err)
 
 		txnClient := env.PachClient.WithTransaction(txn)
 
+		_, err = txnClient.PfsAPIClient.CreateProject(txnClient.Ctx(), &pfs.CreateProjectRequest{
+			Project: client.NewProject(project), // projects is not transactional, so it shouldn't be added to inspect transaction request count
+		})
+		require.NoError(t, err)
```

Comment:
It would be nice to supply a message with each assertion.  `require.NoError(t, err, "CreateProject(%s) should succeed", project)`.